addons/crm/models/crm_team.py

```diff
@@ -155,6 +155,8 @@ def _alias_get_creation_values(self):
             has_group_use_lead = self.env.user.has_group('crm.group_use_lead')
             defaults['type'] = 'lead' if has_group_use_lead and self.use_leads else 'opportunity'
             defaults['team_id'] = self.id
+            if not self.assignment_enabled and self.member_ids:
+                defaults['user_id'] = random.choice(self.member_ids).id
```

Comment:
Nope, this is globally invisible and is going to create a mess people can't debug (aliases are technical).